src/frontend/csrs-portal/src/app/components/landing/landing.component.ts

```diff
@@ -82,7 +71,4 @@ export class LandingComponent implements OnInit {
     console.log('inside logout');
```

Comment:
Can we avoid using console, prefer $log https://docs.angularjs.org/api/ng/service/$log#!

Reply:
You mentioned about Angularjs. it is the first version of Angular. We are using Angular 12 for portal development. It means that $log does not exist in Angular 12. It was developed Logger service for this purpose.